fathom/src/pass/surface_to_doc.rs

```diff
@@ -82,43 +82,47 @@ struct ItemMeta {
 }
 
 impl Context {
-    fn from_alias(&self, writer: &mut impl Write, alias: &Alias) -> io::Result<(String, ItemMeta)> {
-        let id = format!("items[{}]", alias.name.data);
+    fn from_constant(
+        &self,
+        writer: &mut impl Write,
+        constant: &Constant,
+    ) -> io::Result<(String, ItemMeta)> {
+        let id = format!("items[{}]", constant.name.data);
 
         writeln!(
             writer,
-            r##"        <dt id="{id}" class="item alias">"##,
+            r##"        <dt id="{id}" class="item constant">"##,
             id = id,
         )?;
-        match &alias.type_ {
+        match &constant.type_ {
             None => writeln!(
                 writer,
                 r##"          <a href="#{id}">{name}</a>"##,
                 id = id,
-                name = alias.name.data,
+                name = constant.name.data,
             )?,
             Some(r#type) => writeln!(
                 writer,
                 r##"          <a href="#{id}">{name}</a> : {type_}"##,
```

Comment:
We should probably add a `const` keyword here:
```suggestion
                r##"          const <a href="#{id}">{name}</a> : {type_}"##,
```
Note that this will require rebuilding the snapshots.

Reply:
Ah yes, good catch, thanks!